app/views/trips/show.html.erb

```diff
@@ -65,8 +65,8 @@
                 <%= image_tag @trip.badge, style:"width:100px;height100px;"%>
               </div>
               <br/>
-              <p >恭喜你!挑戰成功!</p>  
-              <p >已獲得 <%= @trip.points %> 點</p>  
+              <p >恭喜你!已挑戰成功!</p>  
```

Comment:
Better to use i18n for locale string display

Reply:
Yes since gogoro is international brand, please consider foreigners will use this